### ISSUE TITLE
Face recognition (People) in other account

### ISSUE THREAD

**Issue by wtrdk** [hooram/ownphotos-backend#73]
As 'User 1' I have 3 people in my facial recognition.
When I log in as 'User 2' I also see 3 people, but I haven't even scanned the folder yet.
Seems like a bug! ;-)

**Comment by hooram** [comment]
Good point, the line in question is https://github.com/hooram/ownphotos/blob/dev/api/views.py#L583, for future reference.

**Comment by wtrdk** [comment]
Update:
This also happens with 'Place tree'. Logged in as 'User 1' I see the Place Tree of 'User 2'.

**Comment by hooram** [comment]
Actually, has to do with this line https://github.com/hooram/ownphotos/blob/dev/api/api_util.py#L221
